src/components/slide-deck/index.js

```diff
@@ -209,12 +232,14 @@ class SlideDeck extends Component {
             instructions={this.state.instructions}
             isComplete={finished}
             isFullscreen={this.state.isFullscreen}
+            isLikertScale={this.props.assessment && this.props.assessment.scoring_scale === "LIKERT_CUMULATIVE_POMP"}
```

Comment:
Could you only check for the scoring scale here if the first condition is handling loading?

Reply:
Good point +1